### PR TITLE
pin packaging to 21.3

### PR DESCRIPTION
**Pull request by wamserma**
fix the build as discussed in https://github.com/NixOS/nixpkgs/pull/210230#issuecomment-1521237952

### CODE REVIEW

**Comment by wamserma** [comment]
ping @raboof 
Can you pick this up, so we can get the cve-bin-tool bump into 23.05?

File: pkgs/tools/security/cve-bin-tool/default.nix [hunk 1]
```diff
@@ -30,7 +30,42 @@
...
+# pinned packaging
+, pyparsing
+, fetchPypi
+, buildPythonPackage
+, pretend
+, pythonOlder
 }:
+
+let
+  # pin packaging to < 22 until issue related to https://github.com/intel/cve-bin-tool/pull/2436 are resolved by upstrem
```
Comment (raboof) [inline pkgs/tools/security/cve-bin-tool/default.nix:42]: Can you add a comment in which version of cve-bin-tool this pin can be removed?

**Review by raboof — COMMENTED** [review]
Workaround is fine for me.

Can you edit the PR and commit message to start with `cve-bin-tool: `?

File: pkgs/tools/security/cve-bin-tool/default.nix (hunk 1 above)
Comment (wamserma) [reply pkgs/tools/security/cve-bin-tool/default.nix:42]: there is no upstream milestone, so the best we can do is watch the issue

**Pull request merged**